api/types.go

```diff
@@ -12,21 +15,31 @@ type ProxyConfig struct {
 }
 
 type OIDCConfig struct {
-	IssuerUrl    *string
-	ClientId     *string
-	ClientSecret *string
-	Audience     *string
+	IssuerUrl    *string `json:"issuer_url"`
```

Comment:
For the attributes that are part of the OIDC file,  are all of them required? does it make sense to default some of them making them non required by the user?